graphql_compiler/scripts/generate_test_sql/utils.py

```diff
@@ -0,0 +1,68 @@
+import random
+from uuid import UUID
+
+import six
+
+
+random.seed(0)
```

Comment:
`random.seed()` calls should not be hit on import. Importing a file should not alter global state other than to put the functions and variables of that file into scope.